### ISSUE TITLE
Incorrect indent errors when breaking out of PHP inside an IF statement

### ISSUE THREAD

**Issue by easdue** [squizlabs/PHP_CodeSniffer#1958]
I'm using PHP_CodeSniffer version 3.2.3 on Fedora 27 and when I break out of PHP inside an IF statement I get incorrect indent errors when subsequently continuing php. 

```
<?php
$i = 2;

if ($i < 5) {
?>
    <table border="0" cellspacing="0">
        <tr>
            <td>Dummy table</td>
        </tr>
    </table>
<?php
    if ($i < 2) {
        echo "What's going on here?";
    }
}
```

In the sample code above I get [phpcs] Line indented incorrectly; expected 0 spaces, found 4 on the second if statement and the closing bracket

**Comment by gsherwood** [comment]
This is because the ScopeIndent sniff uses your open tag placements to set the base indent for that code block. You've placed the open PHP tag back at column 1, so it thinks you want to start that block at column 1 as well.

This also allows for code blocks where a control structure is indented way too far to the right, but only to fit in with the HTML blocks around it:
```php
<?php
$i = 2;

if ($i < 5) {
    ?>
    <table border="0" cellspacing="0">
        <tr>
            <td>Dummy table</td>
            <?php
            if ($i < 2) {
                echo "<td>What's going on here?</td>";
            }
            ?>
        </tr>
    </table>
    <?php
}
```
So if you don't want errors with your code block example, you'll need to indent the open tag one more level so it lines up with the code block that it is starting. There is no option to change this behaviour; it's just how the scope indent sniff works.

Does that explain things?

**Comment by easdue** [comment]
Ok, got it.

Thanks for the quick response.
